folly/Portability.h

```diff
@@ -205,7 +205,7 @@ FOLLY_GCC_DISABLE_NEW_SHADOW_WARNINGS
  */
 #if defined(_MSC_VER)
 #define FOLLY_TLS __declspec(thread)
-#elif defined(__GNUC__) || defined(__clang__)
+#elif (defined(__GNUC__) || defined(__clang__))
```

Comment:
Revert this? It's not related to the PR.

Reply:
typo from when I removed a previous change that was no longer necessary, good catch!